docker/hive/Dockerfile

```diff
@@ -0,0 +1,100 @@
+# trunk-ignore-all(terrascan/AC_DOCKER_0047)
+
+# CairoVM Chain
+FROM ghcr.io/dojoengine/dojo:v0.5.1 as katana
+
+# Indexer service
+### Apibara DNA indexer and indexer
+FROM quay.io/apibara/starknet:1.2.0 as apibara
+FROM quay.io/apibara/sink-mongo as indexer
+FROM debian:bookworm as apibara-build
+RUN apt-get update && apt-get install -y patchelf && rm -rf /var/lib/apt/lists/*
+# Run `docker image inspect apibara/starknet:1.2.0` to get the exact path
+COPY --from=apibara /nix/store/swzxd9vbhhj92qzh16j48vb011q57v59-apibara-starknet-1.2.0/bin/apibara-starknet /usr/local/bin/starknet
+COPY --from=indexer /nix/store/3iqnrcirqpg4s7zdy1wdh0dq17jwzmlc-apibara-sink-mongo-0.5.3/bin/apibara-sink-mongo /usr/local/bin/sink-mongo
+# Change the interpreter path.
+# x86-64  "/lib64/ld-linux-x86-64.so.2"
+# aarch64 "/lib/ld-linux-aarch64.so.1"
+RUN patchelf --set-interpreter /lib/ld-linux-aarch64.so.1 /usr/local/bin/starknet && \
```

Comment:
This probably needs to be improved in order to update the interpreter path based on the arch...

Reply:
why do we hard code here something like to arm?